core/src/test/java/amino/run/samplepolicy/TestPolicy.java

```diff
@@ -0,0 +1,17 @@
+package amino.run.samplepolicy;
+
+import amino.run.policy.DefaultPolicy;
+
+public class TestPolicy extends DefaultPolicy {
+    public static class ServerPolicy extends DefaultServerPolicy {
+        public static String getPolicyName() {
```

Comment:
Why do you make this test class so unnecessarily complicated and confusing?  What has getPolicyName got to do with anything?
  
Can't these methods just be called "staticMethod" and return a simple value like "true"?

Reply:
Modified method name to "staticMethod".